digiwf-cypress/readme.md

```diff
@@ -0,0 +1,22 @@
+# Using Cypress tests
+
+## Getting started
+
+Download and install cypress like described in: https://docs.cypress.io/guides/getting-started/installing-cypress
```

Comment:
Reicht zum Installieren nicht ein `npm install` in dem Cypress-Verzeichnis. Für das ist ja die `package.json` da